docusaurus/docs/reactnative/guides/channel_background_customization.mdx

```diff
@@ -0,0 +1,318 @@
+---
+id: channel-background-customization
+sidebar_position: 13
+title: Channel Background Customization
+---
+
+You can implement this customization in your existing app or create a new one by running the following
+
+```shell
+npx react-native init MyApp --template stream-chat-react-native-template
+```
+
+## Basic Custom Background
+
+You can change the background statically by simply wrapping `MessageList` and `MessageInput` by using the `ImageBackground` component.
+
+<img
+  src='https://user-images.githubusercontent.com/25864161/167857632-c0bc9d67-0a84-4cf5-9d75-305e3bcd1f3d.png'
+  width='320px'
+/>
+
+Also make sure you adjust the `theme` correctly.
+
+```tsx
+import { Channel, MessageInput, MessageList, ThemeProvider } from 'stream-chat-react-native';
+import { ImageBackground } from 'react-native';
+
+const IMAGE_URI =
+  'https://images.unsplash.com/photo-1549125764-91425ca48850?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8NjF8fHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=800&q=60';
+
+const ChannelScreen = ({ channel }) => (
+  <ThemeProvider style={theme}>
```

Comment:
where does the `theme` come from? we could add a small example theme for a bit more clarity

Reply:
from line 47, I've moved to the top, so it'll be more clear